terraform/staging/main.tf

```diff
@@ -244,11 +262,22 @@ module "front_door" {
 module "networking" {
   source = "../modules/networking"
 
+  providers = {
+    aws.eu-west-1 = aws.eu-west-1
+    aws.eu-west-3 = aws.eu-west-3
+    aws.us-east-1 = aws.us-east-1
+  }
+
   prefix                                  = local.prefix
   vpc_cidr_block                          = "10.0.0.0/16"
   vpc_flow_cloudwatch_log_expiration_days = 60
 }
 
+moved {

```

Comment:
I'll remove this at the point of merging, I just wanted to make clear that I'd done this (it's already been applied).